src/wp-includes/ms-network.php

```diff
@@ -107,35 +108,44 @@ function clean_network_cache( $ids ) {
  * cache using the network group with the key using the ID of the networks.
  *
  * @since 4.6.0
+ * @since 6.1.0 Introduced the `$update_meta_cache` parameter.
  *
  * @param array $networks Array of network row objects.
+ * @param bool  $update_meta_cache Whether to update sitemeta cache. Default true.
  */
-function update_network_cache( $networks ) {
+function update_network_cache( $networks, $update_meta_cache = true ) {
 	$data = array();
 	foreach ( (array) $networks as $network ) {
 		$data[ $network->id ] = $network;
 	}
+
 	wp_cache_add_multiple( $data, 'networks' );
+	if ( $update_meta_cache ) {
+		$network_ids = array_keys( $data );
+		update_meta_cache( 'site', $network_ids );
+	}
 }
 
 /**
  * Adds any networks from the given IDs to the cache that do not already exist in cache.
  *
  * @since 4.6.0
+ * @since 6.1.0 Introduced the `$update_meta_cache` parameter.
  * @access private
  *
  * @see update_network_cache()
  * @global wpdb $wpdb WordPress database abstraction object.
  *
  * @param array $network_ids Array of network IDs.
```

Comment:
These descriptions should also be re-aligned.

Reply:
Fixed